src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -71,6 +71,22 @@ namespace chip {
 #define CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT (200_ms32)
 #endif // CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT
 
+/**
+ *  @def CHIP_CONFIG_MRP_ON_FIRST_MSG_FAIL_DO_MDNS_LOOKUP
+ *
+ *  @brief
+ *    Should an address lookup of the peer happen on every first message that fails
+ *    to send on the link.
+ *
+ *  The default value selected to not perform lookup was for the following reasons:
+ *    1. The likelihood of the IP address changing right away is extremely slim.
```

Comment:
Not sure what 'right away' is relative to.

```suggestion
 *    1. There is insufficient empirical data to indicate the frequency of occurrence of IP addresses changing *without* a reboot. Doing this to benefit a possibly small set of use-cases while causing harm in what is a fairly common scenario of encountering degraded networking conditions doesn't seem justified.
```

Reply:
This could come after point 2) below.